src/types/hexadecimal.js

```diff
@@ -0,0 +1,93 @@
+import * as validate from './validate'
+
+class BaseHexType {
```

Comment:
The only case is to override implementation like I did with Uuid type. See comment below.

As long as there're the classes elsewhere I thought it would be appropriate to keep them; I'll update the code in next commit.